index.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title> Delicious | THE BEST FOOD BLOG ON THE WEB </title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <!-- Place favicon.ico in the root directory -->
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+
+    <nav>
+      <img class="banner-logo" src="http://i.imgur.com/5tFwpoN.png">
+      <h1> Delicious <br> THE BEST FOOD BLOG ON THE WEB </h1>
+      <ul>
+        <li> <img src="http://i.imgur.com/Xz0logE.png"> </li>
+        <li> <img src="http://i.imgur.com/FnlhJVH.png"> </li>
+        <li> <img src="http://i.imgur.com/pfmEjMI.png"> </li>
+        <li> <img src="http://i.imgur.com/ylgYf41.png"> </li>
+        <li> <img src="http://i.imgur.com/zSxL2IW.png"> </li>
+        <li> <img src="http://i.imgur.com/vCWEjwG.png"> </li>
+        <li> <img src="http://i.imgur.com/wxiFpvM.png"> </li>
+        <li> <img src="http://i.imgur.com/jrfNOkO.png"> </li>
```

Comment:
Remember these will be links, so going forward wrap the images in an 'a href="#"'